subsys/dfu/src/dfu_target_mcuboot.c

```diff
@@ -12,11 +12,46 @@
 
 LOG_MODULE_REGISTER(dfu_target_mcuboot, CONFIG_DFU_TARGET_LOG_LEVEL);
 
+#define MAX_FILE_SEARCH_LEN 500
```

Comment:
why 500?
and not CONFIG_AWS_FOTA_FILE_PATH_MAX_LEN or user-configurable?

Reply:
This is not related to AWS (can be used directly through http), also this simply dictates the search length, hence it does not affect stack size etc. This is only here to stop the search from taking forever.

Hence I think it is sufficient to hard code a value of 500.

Reply:
oki